core/src/main/java/io/nixer/nixerplugin/core/stigma/token/validation/StigmaTokenPayloadValidator.java

```diff
@@ -1,10 +1,6 @@
 package io.nixer.nixerplugin.core.stigma.token.validation;
 
 import java.text.ParseException;
-import java.time.Duration;
-import java.time.Instant;
-import java.util.Date;
-import java.util.function.Supplier;
```

Comment:
This class is a delegate in another class under JwtValidator interface. It adds complexity and I am not sure we get any benefit of it.

Reply:
I agree - as for stigma providers, I am going to merge both and remove interface in another PR.